plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsContainerOptionsMapper.groovy

```diff
@@ -47,15 +47,15 @@ class AwsContainerOptionsMapper {
             final tokens = value.tokenize('=')
             keyValuePairs << new KeyValuePair().withName(tokens[0]).withValue(tokens.size() == 2 ? tokens[1] : null)
         }
-        if ( keyValuePairs.size() > 0 )
```

Comment:
This is implicit with groovy, have a look at https://groovy-lang.org/semantics.html#Groovy-Truth

Reply:
Yep, sorry, that comes from my Java background :)